### ISSUE TITLE
Remove F5 Hosts when hosts not available

### ISSUE THREAD

**Issue by gwolfis** [f5devcentral/vscode-f5#182]
When using the extension to login to public cloud hosts it happens that those hosts are not available anymore due to automatic deploy and deletion of BIG-IP environments.

The extension refushes to remove F5 Hosts when it is not able to connect to and throws an error.

--
Error running command f5.removeHost: Running the contributed command: 'f5.removeHost' failed.. This is likely caused by the extension that contributes f5.removeHost.
--

**Comment by DumpySquare** [comment]
Thanks.  I will be looking into this when I return from leave next month.

**Comment by DumpySquare** [comment]
Hi @gwolfis.  This should be fixed in some of the recent versions.  Can you confirm?

**Comment by gwolfis** [comment]
Yes, can confirm that it has been solved and removing of hosts works now. Thanks!
